charts/armo-components/values.yaml

```diff
@@ -60,6 +60,9 @@ gkeProject:
 nameOverride: ""
 fullnameOverride: ""
 
+# -- for enable:"<any-value>", for disable:"": the print of json posted to the backend from the vuln scanner 
+PrintVulnScannerPostJSON: ""
```

Comment:
1. Should be lowercase
2. Should be in the relevant component scope

Reply:
1. PrintVulnScannerPostJSON -> printVulnScannerPostJSON?
2. I have added it in the same scope of triggerNewImageScan, are u sure this is not ok?

Reply:
Yes.
1. `PrintVulnScannerPostJSON ` -> `verbose`
2. Yes. This is specifically for that component. The triggering new image scanning was for collector/websocket

Reply:
1. verbose is too general the purpose here is just print the final json sent to the backend, so on your permissions I'll stay it as  printVulnScannerPostJSON(with lower p)

Reply:
I understand, but I still think we should use names that others will understand 
We cant have an env for each different log :)